location_extractor/containers.py

```diff
@@ -2,25 +2,27 @@
 from typing import List, Optional
 import abc
 
+from location_extractor.clients import LocationDTO
+
 
 class Entity(abc.ABC):
     @classmethod
     @abc.abstractmethod
-    def from_dict(cls, **kwargs):
+    def from_dto(cls, dto: LocationDTO):
         """
         Convert dictionary to Entity
-        :param kwargs: values for Entity fields
+        :param dto: LocationDTO
         :return: Entity
         """
 
     @classmethod
-    def from_dicts(cls, dictionaries: List[dict]):
+    def from_dtos(cls, dtos: List[LocationDTO]):
```

Comment:
```suggestion
    def from_dtos(cls, dtos: List[LocationDTO]) -> List['Entity']:
```

Let's add add type hinting for returned values, because we are going to introduce `mypy` to the project and this great tool assume that if there is no type hinting for returned value it set it `Any` (or `None` in some cases like in `__ini__()`